examples/configuration.nix

```diff
@@ -203,5 +211,5 @@
   # The nix-bitcoin release version that your config is compatible with.
   # When upgrading to a backwards-incompatible release, nix-bitcoin will display an
   # an error and provide hints for migrating your config to the new release.
-  nix-bitcoin.configVersion = "0.0.26";
+  nix-bitcoin.configVersion = "0.0.29";
```

Comment:
This should be `0.0.30` as `v0.0.29` was released in the meantime and this PR will be released with `v0.0.30`.

Reply:
Fixed.